lms/static/scripts/frontend_apps/components/LaunchErrorDialog.tsx

```diff
@@ -1,23 +1,22 @@
 import { Link } from '@hypothesis/frontend-shared';
 
+import type { ErrorLike } from '../errors';
+import type { ErrorState } from './BasicLTILaunchApp';
 import ErrorModal from './ErrorModal';
 
-/**
- * @typedef {import("preact").ComponentChildren} Children
- * @typedef {import('./BasicLTILaunchApp').ErrorState} ErrorState
- * @typedef {import('../errors').ErrorLike} ErrorLike
- */
-
-/**
- * @typedef LaunchErrorDialogProps
- * @prop {boolean} busy -
- *   Flag indicating that the app is busy and should not allow the user to
- *   click the "Try again" button
- * @prop {ErrorState} errorState - What kind of error occurred?
- * @prop {ErrorLike|null} error - Detailed information about the error
- * @prop {() => void} onRetry -
- *   Callback invoked when user clicks the "Try again" button
- */
+export type LaunchErrorDialogProps = {
+  /**
+   * Flag indicating that the app is busy and should not allow the user to
+   * click the "Try again" button
+   */
+  busy: boolean;
```

Comment:
I considered taking the opportunity to rename `bussy` to `loading`, however, it has many side effects (tests, components consuming this component, their own tests, etc).

Also, the nested `ErrorModal` component also calls the prop `busy`, making it a bit odd to change `busy={busy}` to `busy={loading}`.

Because of that, it probably makes sense to do it separately.

Reply:
Understood!